examples/kubernetes/snapshot/specs/snapshot-import/README.md

```diff
@@ -0,0 +1,70 @@
+# Volume Snapshots
+
+## Overview
+
+This driver implements basic volume snapshotting functionality using the [external snapshotter](https://github.com/kubernetes-csi/external-snapshotter) sidecar and creates snapshots of EBS volumes using the `VolumeSnapshot` custom resources.
+
+## Prerequisites
+
+1. Kubernetes 1.17+ (CSI 1.0).
```

Comment:
Why do we need 1.17?

Reply:
We are using beta snapshot api, and recording to this article https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-cis-volume-snapshot-beta/, kube version should be 1.17+